src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1640,12 +1640,12 @@ public String getHelpFile(String fieldName) {
             return n;
         }
 
-        public ListBoxModel doFillCredentialsIdItems(@AncestorInPath ItemGroup context,
+        public ListBoxModel doFillCredentialsIdItems(@AncestorInPath AccessControlled context,
                                                      @QueryParameter String host,
                                                      @QueryParameter String port,
                                                      @QueryParameter String credentialsId) {
-            AccessControlled _context = (context instanceof AccessControlled ? (AccessControlled) context : Jenkins.getInstance());
-            if (_context == null || !_context.hasPermission(Computer.CONFIGURE)) {
+            Jenkins jenkins = Jenkins.getActiveInstance();
+            if ((context == jenkins && !jenkins.hasPermission(Computer.CREATE)) || (context != jenkins && !context.hasPermission(Computer.CONFIGURE))) {
```

Comment:
This might result in a regression on Jenkins before 2.78 where it no longer is enough to have system-wide `Computer.CONFIGURE` to get credentials.